### PR TITLE
Add an article about recent libraries deprecation.

### PR DESCRIPTION
**Pull request by shulard**
We choose to drop unused, unpopular and not relevant part of Hoa and it's important to write about that.

Related to : hoaproject/Central#90

It's a quick and simple article, maybe we need to add more details ?

### CODE REVIEW

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<?xyl-overlay href="hoa://Application/Overlays/Article.xyl"?>
+<?xyl-meta name="title" value="Now, it's time to deprecate unused libraries."?>
+<?xyl-meta name="date" value="2018-06-15T19:00:18+01:00"?>
+
+<overlay xmlns="http://hoa-project.net/xyl/xylophone">
+  <article id="main">
+    <p>Hoa is a set of 61 PHP libraries, during our <a href="http://discourse.hoa-project.net/t/hoa-virtual-meeting/22/34">latest
+    Virtual Meeting</a> we choose to deprecate some libraries. This is important
+    for us to ensure that Hoa stays relevant and up to date regarding real
+    world needs.<br/>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]: You can remove the `<br />`.

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    We choose to remove the ones that are:</p>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- We choose to
+ We have decided to
```

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <ol>
+      <li>Un-used or unpopular,</li>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]: “Unpopular” in this context means libraries that people don't like. This is _wrong_ in some regards.

I would rephrase like this:

> Basically un-used (with less than 100 downloads in 5 years).

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <li>Not relevant anymore because they refer to dead technologies.</li>
+    </ol>
+    <p>The main goal is to reduce the effort to maintain the Hoa ecosystem.</p>
+    <p>To be sure about the list, we checked some packagist and github stats.
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- the list
+ the list items
```

Comment (Hywan) [reply Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- some packagist and github stats
+ Packagist and GitHub statistics
```

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    We also choose to keep the one used internally at the moment.</p>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- the one used internally
+ the ones we use for private projects
```

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    <p>Some of the libraries weren't registered on packagist which means that
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- packagist
+ Packagist

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    they are not used in today PHP's world.</p>
+    <p>The deprecation process is simple:</p>
+    <ol>
+      <li>Abandon the project on Packagist,</li>
+      <li>Update the README.md file to inform users,</li>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- README.md
+ <code>README.md</code>
```

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+      <li>Archive the repository on Github.</li>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]: You can add that mirrors are also updated to reflect this change.

File: Source/Posts/2018/06-deprecate-unused-libraries.xyl [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    </ol>
+    <p>Here is the whole list of deprecated libraries:</p>
+    <ul>
+      <li><a href="https://github.com/hoaproject/cache">hoa/cache</a></li>
```
Comment (Hywan) [inline Source/Posts/2018/06-deprecate-unused-libraries.xyl]:
```diff
- hoa/cache</a>
+ <code>hoa/cache</code></a>,
```

etc.

**Review by Hywan — CHANGES_REQUESTED** [review]
Despites some modifications, the post looks good to me. It is clear, and we understand the motivations. Thanks your very much!

**Comment by shulard** [comment]
Hello @Hywan, I've updated the article with all your feedbacks 😄.
